addons/website/static/src/scss/website.scss

```diff
@@ -199,6 +199,11 @@ font[class*='bg-'] {
         border-top-left-radius: $size;
         border-bottom-right-radius: $size;
     }
+    &.rounded-empty-circle {
+        @extend .rounded-circle;
+        border: (2 * $border-width) solid $gray-400;
```

Comment:
@rdeodoo Maybe a bit too violent? Maybe `ceil(1.4 * $border-width)`? Or am I overthinking this?

Reply:
As you prefer, I don't see what was wrong with `2px` anyway as whatever theme you use you most likely won't want a thinner/bigger border in this case, so I might as well do that.

![Screenshot from 2019-12-27 16-01-23](https://user-images.githubusercontent.com/30048408/71521722-2f446780-28c2-11ea-8e3e-da86a85148ef.png)

Reply:
Mmh, maybe you are right, border-width are often difficult to have an opinion on... but if a theme is stupid enough to have border-width: 5px... I think the logic would be to use bigger width here too, no? but as the size is also hardcoded... don't know.